examples/powertools-examples-core/gradle/build.gradle

```diff
@@ -11,10 +11,6 @@ wrapper {
 compileJava {
     sourceCompatibility = "1.8"
     targetCompatibility = "1.8"
-
-    ajc {
```

Comment:
@scottgerring FYI, I think this is not needed as you already use `aspect`

Reply:
If it works without it let's get rid of it!